modules/teams/components/user-selection.tsx

```
@@ -0,0 +1,189 @@
+      )}
+
+      {selectedUsers.length > 0 && (
+        <div className="space-y-4">
+          <div className="flex items-center justify-between">
+            <h3 className="text-sm font-medium text-gray-700">
+              Selected Teammates ({selectedUsers.length})
+            </h3>
+            <button
+              onClick={handleClearAllUsers}
+              className="text-xs font-medium text-red-600 transition-colors hover:text-red-800"
+            >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)